### PR TITLE
Lets elaborate

### PR DESCRIPTION
**Pull request by nelsonic**
Adds some detail.

### CODE REVIEW

**Comment by nelsonic** [comment]
@iteles I'm going to add more to this. but will do more PRs as and when.
Please review/merge *this iteration*. thanks.

File: README.md [hunk 1]
```diff
@@ -7,45 +7,71 @@ If you have ***any questions*** please
...
-We ***craft code*** to ***scratch our own itch*** and ***everything*** we do is ***always Open Source***
+### For Us *By* Us
+
+We ***craft code*** to [***scratch our own itch***](https://github.com/dwyl/start-here#our-approach-scratching-your-own-itch) and ***everything*** we do is ***always Open Source***
 
 | Project | Use Case | Build | Coverage | Dependencies | Learn |
 | --------|----------|:-----:|:--------:|:------------:|-------|
 | **[hapi-auth-jwt2]** | Authorization | [![build][jwt2-bi]][jwt2] | [![cov][jwt2-cc]][jwt2] | [![deps][jwt2-dep]][jwt2] | [learn-jwt] |
-| **[env2]** | Environment Variables | [![build][env2-bi]][env2] | [![cov][env2-cc]][env2] | [![deps][env2-dep]][env2] | [learn-env2] |
+| **[env2]** | Environment Variables | [![build][env2-bi]][env2] | [![cov][env2-cc]][env2] | [![deps][env2-dep]][env2] | [learn-env] |
```
Comment (iteles) [inline README.md]: Dependencies badges appear to be broken in the view:
broken dependencies links

Comment (nelsonic) [inline README.md]: did you try clicking on them to see that david-dm is ***down***...?

Comment (nelsonic) [inline README.md]: The current readme hides the "broken" images:

**Pull request merged**